src/Raven.Client/Documents/Commands/Batches/DeleteCommandData.cs

```diff
@@ -9,24 +9,38 @@ namespace Raven.Client.Documents.Commands.Batches
     public class DeleteCommandData : ICommandData
     {
         public DeleteCommandData(string id, string changeVector)
+            :this(id, changeVector, changeVector)
+        {
+            
+        }
+        public DeleteCommandData(string id, string changeVector, string oldChangeVector)
         {
             Id = id ?? throw new ArgumentNullException(nameof(id));
             ChangeVector = changeVector;
+            OldChangeVector = oldChangeVector;
```

Comment:
I'm open for a new one.
Basically, this is the change vector that was read by the client, we need to send it to the server, but we can't use `ChangeVector`, since that isn't optimistic concurrency check.

Reply:
OriginalChangeVector? ClusterChangeVector? Not sure...